src/main/java/net/glowstone/entity/GlowEntity.java

```diff
@@ -534,6 +611,25 @@ public void setRawLocation(Location location) {
     public abstract List<Message> createSpawnMessage();
 
     /**
+     * Creates a {@link Message} which can be sent to a client directly after the entity is spawned.
```

Comment:
In this case it would represent a *List* of messages to be sent, not a single message.